certified-connectors/Africa's Talking Payments/apiDefinition.swagger.json

```diff
@@ -1208,4 +1208,4 @@
     }
   ],
   "tags": []
-}
\ No newline at end of file
+}
```

Comment:
I still see a change, seems like a newline?

Reply:
I checked their Definition, and it shows it only has 1211 lines. shouldnt I just keep it? I added a line so its 1212, but i think i need to revert back.

Reply:
Yeah, we need to make sure it doesn't show as a change when we make the PR to Microsoft's repo.
If nothing else works, you could download the file from https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/Africa's%20Talking%20Payments/apiDefinition.swagger.json and replace the one you have.